front/src/views/exportBankExtractComponent/index.jsx

```diff
@@ -57,9 +65,23 @@ export default class BankExtractModal extends React.Component {
     }
 
     readCSVCaixa = () => {
```

Comment:
Se a única diferença entre essa função readCSVCaixa e a função readCSVBB for os delimitadores, cria uma única função que recebe o delimitador como parâmetro. Vai ficar melhor o código.